bigquery/indexer.py

```diff
@@ -80,24 +80,31 @@ def index_table(es, index_name, primary_key, table_name, billing_project_id):
                          (primary_key, table_name))
 
     start_time = time.time()
+
     # Use generator so we can index large tables without having to load into
     # memory.
-    k = (
-        {
-            '_op_type': 'update',
-            '_index': index_name,
-            # type will go away in future versions of Elasticsearch. Just use any string
-            # here.
-            '_type': 'type',
-            '_id': row[primary_key],
+    def es_actions(df):
```

Comment:
I'm not sure what es_actions means, maybe a more specific name would help?

Reply:
Anyone who's familiar with Elasticsearch client library will know this, see https://elasticsearch-py.readthedocs.io/en/master/helpers.html